test/unit/controllers/declaration/SupervisingCustomsOfficeControllerSpec.scala

```diff
@@ -0,0 +1,199 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package unit.controllers.declaration
+
+import controllers.declaration.SupervisingCustomsOfficeController
+import helpers.views.declaration.WarehouseIdentificationMessages
+import models.{DeclarationType, Mode}
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito
+import org.mockito.Mockito.{verify, when}
+import org.scalatest.{BeforeAndAfterEach, OptionValues}
+import play.api.libs.json.Json
+import play.api.test.Helpers._
+import play.twirl.api.HtmlFormat
+import unit.base.ControllerSpec
+import views.html.declaration.supervising_customs_office
+
+class SupervisingCustomsOfficeControllerSpec extends ControllerSpec with BeforeAndAfterEach with WarehouseIdentificationMessages with OptionValues {
+
+  val supervisingCustomsOfficeTemplate: supervising_customs_office = mock[supervising_customs_office]
+
+  val controller = new SupervisingCustomsOfficeController(
```

Comment:
We have some fields marked as `private` and some not. Lets be consistent :) 
They probably all want to be `private`